pkg/ivd/ivd_protected_entity_test.go

```diff
@@ -776,3 +779,13 @@ func setupPETM(typeName string, logger logrus.FieldLogger) (*s3repository.Protec
 	}
 	return s3petm, err
 }
+
+func TestCheckConnectionResetError(t *testing.T) {
```

Comment:
It might be better to place this test case in utils_test.go (if no, create one) under pkg/util/

Reply:
Move to utils_test.go